index.js

```diff
@@ -1 +1,85 @@
 // implement your API here
+
+const express = require('express');
+const db = require('./data/db.js');
+
+const server = express();
+server.use(express.json());
+
+server.get('/api/users', (req, res) => {
+  db.find()
+    .then(users => {
+      res.status(200).json(users);
+      console.log(users);
+    })
+    .catch(err => {
+      res.status(500).json({
+        error: 'The users information could not be retrieved.'
+      });
+    });
+});
+
+server.post('/api/users', (req, res) => {
+  const { name, bio } = req.body;
+  const newUser = {
+    name,
+    bio
+  };
+
+  db.insert(newUser)
```

Comment:
Here you can add some validation to check to see if our request body has a `"name"` and `"bio"`. This will ensure that we don't get `null` back from the server. 👍